### ISSUE TITLE
 attribute table - Possibility to override display field label

### ISSUE THREAD

**Issue by Gaetanbrl** [georchestra/mapstore2-georchestra#637]
## Description

With the aim of display complexe attributes for many users profils, some users asked to rename many fiels.
This seems not possibile in mapstore config (attribute table plugin).

With this enhancement, it could be possible to display a label for a specific attribute/field name to ease user understanding.

Here an example to rename `"s_r0.6"` field name by `"scenario_0.6"` with new  `displayLabel` config property (could be another prop name) : 

```
{
  "name": "FeatureEditor",
  "cfg": {
    "maxZoom": 21,
    "customEditorsOptions": {
      "rules": [
        {
          "regex": {
            "attribute": "s_r0.6",
            "url": "regex to match a specific url",
            "typeName": "layerName"
          },
          "displayLabel": "scenario_0.6"
        }
      ]
    }
  }
}
```
@offtherailz @tdipisa Is it a complicated / acceptable contribution ? I will create a dedicated ticket if acceptable.

**What do you thing about this enhancement ?**

**What kind of improvement you want to add?** (check one with "x", remove the others)
- [ ] Minor changes to existing features
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Mapstore2georchestra improvement 
- [ ] Mapstore2 improvement
- [ ] Other... Please describe:

**Comment by Gaetanbrl** [comment]
Maybe this is already possible with label ?

https://github.com/geosolutions-it/MapStore2/blob/3e1d020e5799de91d61a141e2bae6835456ac13b/web/client/plugins/FeatureGrid.jsx#L42

Not sur becayse i don't find anytyhing about that in the MS2 doc : 

https://mapstore2.geosolutionsgroup.com/mapstore/docs/api/plugins#plugins.FeatureEditor

**Comment by tdipisa** [comment]
@Gaetanbrl as far as I understand you need [this](https://docs.mapstore.geosolutionsgroup.com/en/latest/user-guide/layer-settings/#fields) that will be available starting from v2023.02.00

**Comment by Gaetanbrl** [comment]
thanks @tdipisa, It's a great feature !

Just to undestand, if we change an alias for a random field, the attribute table (featureEditor) will display this alias into header col label ?

**Comment by tdipisa** [comment]
@Gaetanbrl yes, the alias is used in the whole application (eg. stile editor, filter layer and so on) where the attribute name is displayed somehow.
